src/core/lombok/bytecode/PoolConstantsApp.java

```diff
@@ -66,7 +66,7 @@
 		CmdReader<CmdArgs> reader = CmdReader.of(CmdArgs.class);
 		CmdArgs args;
 		try {
-			args = reader.make(raw.toArray(new String[0]));
+			args = reader.make(raw.toArray(new String[raw.size()]));
```

Comment:
It can depend of the JVM.
In the second article, you can see the response of the creator of JMH